projects/components/src/header/page/page-header.component.ts

```diff
@@ -22,7 +22,7 @@ import { NavigableTab } from '../../tabs/navigable/navigable-tab';
         <div class="title" *ngIf="this.titlecrumb$ | async as titlecrumb">
           <ht-icon
             class="icon"
-            *ngIf="titlecrumb.icon"
+            *ngIf="titlecrumb.icon && breadcrumbs.length <= 1"
```

Comment:
This looks good and makes sense for almost all screens - the only one I'm hesitating on is backends, which have a different and more useful icon (it's specific to the backend type, which we don't know at the page level where we use the generic backend icon) at the breadcrumb level.